src/others/contexts/filter.tsx

```diff
@@ -92,10 +92,18 @@ export const FilterContextProvider: React.FunctionComponent = ({ children }) =>
     [setFilters]
   );
 
-  const getActiveFilterItems = (filterName: string) => {
+  const getActiveFilterItems = (filterName: string, field?: keyof FilterItem) => {
```

Comment:
**question**

Why csv/json requires changes to filters? I think the filters need a separate refactor (the goal would be to use `keyof AidRequest` as the only accepted filter value. 

Downloading would then require mapping filtered aid request to given format and handling the mechanics.